client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -45,36 +50,35 @@ const ConnectedSingleSiteJetpack = connectOptions(
 		const {
 			analyticsPath,
 			analyticsPageTitle,
+			canManage,
 			emptyContent,
+			filter,
 			getScreenshotOption,
+			hasJetpackThemes,
 			showWpcomThemesList,
 			search,
-			site,
 			siteId,
-			wpcomTier,
-			filter,
-			vertical
+			vertical,
+			wpcomTier
 		} = props;
 		const jetpackEnabled = config.isEnabled( 'manage/themes-jetpack' );
 
 		if ( ! jetpackEnabled ) {
 			return (
 				<JetpackReferrerMessage
```

Comment:
We'll be able to drop this component altogether once we drop the (now obsolete) `manage/themes-jetpack` feature flag.